app/core/migrations/0001_initial.py

```diff
@@ -94,12 +95,34 @@ class Migration(migrations.Migration):
                 ('display_name', models.CharField(help_text='Enter the display name of the sorting option', max_length=200)),
                 ('field_name', models.CharField(help_text='Enter the metadata field name as displayed in Elasticsearch e.g. course.title', max_length=200)),
                 ('active', models.BooleanField(default=True)),
-                ('highlight_icon', models.CharField(choices=[('clock', 'clock'), ('hourglass', 'hourglass'), ('user', 'user'), ('multi-users', 'multi-users'), ('location', 'location'), ('calendar', 'calendar')], default='user', max_length=200)),
+                ('highlight_icon', models.CharField(choices=[('clock', 'clock'), ('hourglass', 'hourglass'), ('user', 'user'), ('multi_users', 'multi_users'), ('location', 'location'), ('calendar', 'calendar')], default='user', max_length=200)),
                 ('rank', models.IntegerField(default=1, help_text='Order in which highlight show on the course detail page (2 items per row)', validators=[django.core.validators.MinValueValidator(1, 'rank shoud be at least 1')])),
                 ('xds_ui_configuration', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='course_highlights', to='core.xdsuiconfiguration')),
             ],
             options={
                 'abstract': False,
             },
         ),
+        migrations.CreateModel(
+            name='XDSUser',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('password', models.CharField(max_length=128, verbose_name='password')),
```

Comment:
Can we ensure that the passwords are hashed and not stored in clear text?

Reply:
Yes, from my understanding. Django Auth handles all that. Ill confirm today and make any changes required.

Reply:
Just validated that the passwords are indeed hashed

Reply:
Excellent, thanks!